operations/rack_resiliency/rack_resiliency.md

```diff
@@ -0,0 +1,489 @@
+
+To address these issues, the Rack Resiliency feature is introduced to provide management rack level resiliency by providing methods and functionality required to maintain the high availability of critical management services due to unplanned outages. This feature will ensure that CSM can tolerate failures of racks and prevent a system wide outage, where such outage encompasses the inability for completing current jobs or scheduling new ones. This feature doesn't address planned downtime of racks for the current release (CSM 1.7.0).
+
+A similar problem emerges for utility storage on management storage nodes. Even though these are not orchestrated by `Kubernetes', incorrect physical placement and/or a lack of correct software configuration, specifically around replication, may cause utility storage service disruptions due to rack-level failures.
+
+### Key features
+
+* Validates and reports node placement configuration in order for admin to reconfigure, if required, where critical management services can tolerate rack failures.
+* Provides tolerance for a single rack failure within the Management Plane Failure Domain (MPFD), ensuring that it does not lead to a system-wide outage.
+* Ensures load balancing and high availability of critical services essential for the seamless execution and completion of user jobs.
+* Monitor and report failures of management node(s) and rack(s) which are part of MPFD
+* Monitor and report critical services health and their balance across racks
+* Minimize operational complexity for admins based on above key features
+* Aligns with future plans for similar functionality in next generation systems management solutions
+* Ease of deploying and managing
+  
+## RR solution overview
+
+The architecture for providing a Rack Resiliency solution includes the following 4 stages, along with a CLI for managing RR.
+
+* Management plane placement discovery and placement validation
+* Spreading of NCNs across `Kubernetes` zones, and spreading storage nodes across `Ceph` zones 
+* Distribute critical services across zones
+* Monitor failures and ensure critical services are available across zones
+* RR CLI
+
+![Rack Resiliency solution overview](../../img/rack-resiliency-1.png)
+
+###  1 Discover management plane and validate for resiliency 
+
+#### 1.1 Identify racks and management/ managed nodes across racks

```

Comment:
This header mentions management and managed both, but all of the text in the section only talks about management.